boards/px4/fmu-v3/default.cmake

```diff
@@ -8,6 +8,8 @@ px4_add_board(
 	LABEL default
 	TOOLCHAIN arm-none-eabi
 	ARCHITECTURE cortex-m4
+	CHIP_MANUFACTURER stm
+	CHIP stm32f4
```

Comment:
We should be able to get this from the NuttX config.
 - CONFIG_ARCH_CHIP
 - CONFIG_ARCH_CHIP_STM32
 - CONFIG_STM32_STM32F4XXX

Reply:
https://github.com/PX4/Firmware/blob/f3eafdc296a8b0413283916ad80450b668b027e4/src/drivers/boards/common/CMakeLists.txt#L44-L48

Reply:
why not add_subdirectory(${CONFIG_ARCH_CHIP})

Reply:
The issue with this is that `CONFIG_ARCH_CHIP` gives `stm32` and not `stm32f4`. I did not see anything that would provide the stm32f4, but maybe I missed it?

Reply:
> but maybe I missed it?
@bkueng - welcome to the world of compressed configs.  Easy to compare but suffer from information hiding. 

You have to look in the .config or config.h file there will be something in the form CONFIG_STM32_STM32F4XXX=y  the we will have to translate and concatenate

Reply:
As discussed with David, personally I generally like to infer this as well, but we both think it's better here not to, because:
- It's not generic for other platforms
- We would need to have NuttX-config specific if-else cases to translate to our directories and extend it every time for a new arch/chip.
- Stating the arch/chip explicitly makes it easier to see what code is used.

@dagar what do you think?

Reply:
Alright, changed.